samtranslator/internal/schema_source/aws_serverless_function.py

```diff
@@ -618,6 +618,7 @@ class Properties(BaseModel):
     SnapStart: Optional[SnapStart] = prop("SnapStart")
     RuntimeManagementConfig: Optional[RuntimeManagementConfig] = prop("RuntimeManagementConfig")
     Tags: Optional[Tags] = prop("Tags")
+    PropagateTags: Optional[bool]  # TODO: add docs
```

Comment:
When `PropagateTags` is set without setting `Tags`, do we fail the transform or just ignore the property?

Reply:
I believe the behaviour should be ignoring the property. I will add a test covering this scenario.